pkg/processor/procbuiltin/extractfield_test.go

```diff
@@ -135,7 +137,7 @@ func TestExtractFieldKey_Process(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			underTest, err := ExtractFieldKey(tt.config)
-			assert.Ok(t, err)
+			is.NoErr(err)
```

Comment:
The `t` in this test is a different `t` from one given here: `func TestExtractFieldKey_Process(t *testing.T)`. Hence, it would be better to initialize `is` in this subtest.